circle.yml

```diff
@@ -1,4 +1,17 @@
 machine:
   node:
     version: 4.3.0
-
+test:
+  override:
+    - ./node_modules/.bin/eslint --quiet:
```

Comment:
http://stackoverflow.com/questions/11580961/sending-command-line-arguments-to-npm-script